Source/Extensions/Blazorise.DataGrid/Utils/Reflection.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Blazorise.DataGrid.Utils
+{
+    public static class Reflection
+    {
+        /// <summary>
+        /// Inits a complex object and inner public instance properties.
+        /// Ignores lists, valuetypes and strings
+        /// </summary>
+        /// <typeparam name="TItem"></typeparam>
+        /// <returns></returns>
+        public static TItem InitObject<TItem>()
```

Comment:
Name it CreateInstance so it is along the Activator naming convention.

Reply:
Sure,